### ISSUE TITLE
How to use this project

### ISSUE THREAD

**Issue by m0re5ecsc4** [sourcegraph/zoekt#784]
The readme of this project has not been updated for a long time. I am new to this project. How should I run it? I can only know by looking at the code. Please update the usage documentation.

**Comment by jtibshirani** [comment]
I agree, we should really update the README! I've added it to our queue.

In the meantime, these usage commands might be helpful: https://github.com/sourcegraph/sourcegraph/blob/462ba3de0bf4fdebfa0ed46f5ee0de3b6182634a/doc/dev/how-to/zoekt_local_dev.md#notes. (These are developer docs from the sourcegraph/sourcegraph repo).

**Comment by 0xku** [comment]
@m0re5ecsc4 the doc folder has examples for indexing a project, for querying you can look at this project https://neogrok-demo-web.fly.dev/ which builds a ui interface on top of zoekt, they have a nice page on querying https://neogrok-demo-web.fly.dev/syntax

**Comment by jtibshirani** [comment]
@m0re5ecsc4 I opened https://github.com/sourcegraph/zoekt/pull/905 to improve the usage examples. Let us know if you have questions still not covered there.

@kuutsav nice, those docs are really approachable! I've added a link to them from our query syntax docs (https://github.com/sourcegraph/zoekt/pull/905/commits/a94469cf5ae3281c7af1d89c84640f5bd8a6167c).
